src/app/Plans/components/PlansTable.tsx

```diff
@@ -265,34 +222,46 @@ const PlansTable: React.FunctionComponent<IPlansTableProps> = ({
             </>
           ),
         },
-        plan.spec.warm ? 'Warm' : 'Cold',
+        isWarmPlan ? 'Warm' : 'Cold',
         {
-          title: isBeingStarted ? (
-            <PlanStatusNavLink plan={plan}>Running - preparing for migration</PlanStatusNavLink>
-          ) : warmState === 'Starting' ? (
-            <PlanStatusNavLink plan={plan}>
-              Running - preparing for incremental data copies
-            </PlanStatusNavLink>
-          ) : !plan.status?.migration?.started || warmState === 'NotStarted' ? (
-            <StatusCondition status={plan.status} />
-          ) : warmState === 'Copying' ? (
-            <PlanStatusNavLink plan={plan}>
-              Running - performing incremental data copies
-            </PlanStatusNavLink>
-          ) : warmState === 'StartingCutover' ? (
-            <PlanStatusNavLink plan={plan}>Running - preparing for cutover</PlanStatusNavLink>
-          ) : (
-            <PlanStatusNavLink plan={plan} isInline={false}>
-              <Progress
-                title={title}
-                value={statusValue}
-                label={statusMessage}
-                valueText={statusMessage}
-                variant={variant}
-                measureLocation={ProgressMeasureLocation.top}
-              />
-            </PlanStatusNavLink>
-          ),
+          title:
+            isBeingStarted && !isWarmPlan ? (
+              <PlanStatusNavLink plan={plan} isInline={false}>
+                <Progress
+                  title={title}
+                  value={statusValue}
+                  label={statusMessage}
+                  valueText={statusMessage}
+                  variant={variant}
+                  measureLocation={ProgressMeasureLocation.top}
+                />
+              </PlanStatusNavLink>
+            ) : isBeingStarted && isWarmPlan ? (
+              <PlanStatusNavLink plan={plan}>Running - preparing for migration</PlanStatusNavLink>
+            ) : isBeingStarted ? (
+              <PlanStatusNavLink plan={plan}>
+                Running - preparing for incremental data copies
+              </PlanStatusNavLink>
+            ) : !plan.status?.migration?.started || planState === 'NotStarted' ? (
+              <StatusCondition status={plan.status} />
+            ) : planState === 'Copying' ? (
+              <PlanStatusNavLink plan={plan}>
+                Running - performing incremental data copies
+              </PlanStatusNavLink>
+            ) : planState === 'StartingCutover' ? (
+              <PlanStatusNavLink plan={plan}>Running - preparing for cutover</PlanStatusNavLink>
+            ) : (
```

Comment:
A couple things about the `isBeingStarted` states:
* I don't think we want to render the <Progress> bar when the plan is being started, it should always show one of the "preparing for ___" states in that case. In fact I think the "cold plan being started" case will be covered by line 239 and you can get rid of the whole case starting on line 228.
* Remember "incremental data copies" is a warm migration thing. You have the isWarmPlan cases backwards, on line 239 you want `!isWarmPlan` instead of `isWarmPlan`. That will make the line 231 case only hit for warm plans which is correct, but if you want to be more explicit there you can also add `&& isWarmPlan` on that line.
* On line 245 I think we should get rid of the `!plan.status?.migration?.started` condition and just use `planState === 'NotStarted'`, we should be in that state if that started property is falsy anyway. See below comment in getPlanState.

We may want to add some new mock plans to show the "starting" state for warm and cold, or at least one or the other.

Reply:
Agree, not having the "starting" state left me having to guess at a few things - I'd love to get one for warm/cold just to be exhaustive. Maybe in a follow-up PR?